src/clj/auth.clj

```diff
@@ -85,12 +85,11 @@
   [request]
   (let [github-data (github-user-data-from-token (extract-token request))]
     {:status 200
-     :headers {"Content-Type" "text/plain"}
-     :session (merge (:session request) github-data)
+     :session (assoc (:session request) :user github-data)
```

Comment:
I really like that.
Better than my `:user-id`, `:user-email` and `:user-name` keys!